lib-rt/CPy.h

```diff
@@ -841,6 +841,28 @@ static PyObject *CPyDict_FromAny(PyObject *obj) {
     }
 }
 
+static PyObject *CPyIter_Next(PyObject *iter)
+{
+    return (*iter->ob_type->tp_iternext)(iter);
+}
+
+static PyObject *CPy_FetchStopIterationValue(void)
+{
+    PyObject *val = NULL;
+    _PyGen_FetchStopIterationValue(&val);
+    return val;
+}
+
+static PyObject *CPyIter_Send(PyObject *iter, PyObject *val)
```

Comment:
This function is a bit strange. Why is `None` special? At least it would be good add a comment.

Reply:
`None` is special here because PEP 380 specifies calling `next(_i)` if the argument is None and `_i.send(arg)` otherwise, in order to work with iterators that are not generators, and so don't have `send`. I'll add a comment.